### PR TITLE
Improve test coverage

### LINKED ISSUE #87: Add tests to improve coverage of loklak.py file
Currently we have `57%` coverage of `loklak.py` file. 
Add more tests for methods written in `loklak.py` file, to expand test coverage.

### PR DESCRIPTION
**Pull request by YJDave**
Try to improving code coverage by adding more tests for functions.

### CODE REVIEW

**Comment by singhpratyush** [comment]
@YJDave: It seems like there is some issue with rebasing this branch. Please follow the best practices and keep your branch in sync with `upstream/master`.

**Comment by YJDave** [comment]
@singhpratyush Thanks for reviewing! 
Updated!

File: tests/test.py [hunk 1]
```diff
@@ -107,13 +120,22 @@ def test_search(self):
...
         """Test aggregations."""
         result = self.loklak.aggregations('fossasia', '2017-01-10',
                                           '2018-01-10', 10)
         data = result.json()
         self.assertEqual(result.status_code, 200)
         self.assertTrue('aggregations' in data)
+
+        result = self.loklak.aggregations()
+        self.assertEqual(result, '{"error": "No Query string has been given to run query for aggregations"}')
     #     self.assertTrue('hashtags' in data['aggregations'])
```
Comment (vaibhavsingh97) [inline tests/test.py]: Have you found solution to include fields? Please see this https://github.com/loklak/loklak_python_api/pull/75#issuecomment-358653952 for more info

Comment (YJDave) [reply tests/test.py]: Not yet, but I will check it out soon.

**Comment by YJDave** [comment]
@daminisatya These all commits have different features/functions/code added, so that in future if somethings goes wrong we can revert the particular commit. Is this that we should only have one commit in one PR? Can you elaborate more? Thanks for reviewing!

**Comment by YJDave** [comment]
@daminisatya Thanks for your advice. I have updated PR as we talked on channel.

**Comment by YJDave** [comment]
@daminisatya ^^

**Pull request merged**